Source/Atomic/UI/UIMenuWindow.h

```diff
@@ -50,6 +51,8 @@ class UIMenuWindow : public UIWidget
 private:
 
     tb::TBSelectItemSource* source_;
+    
+    void FixMenu( tb::TBMenuWindow* ); // get rid of scrollbars for menu popups
```

Comment:
FixMenu is kind of a confusing method name, do we need the method or can this just be called inline from the spot I am seeing it, with a note to remove scroll bars and force a relayout?